charts/argo-cd/values.yaml

```
@@ -1623,6 +1623,11 @@ redis-ha:
     # @default -- See [values.yaml]
     containerSecurityContext:
       readOnlyRootFilesystem: true
+    timeout:
+      # -- HAProxy server connection timeout
+      server: 1860s
+      # -- HAProxy client connection timeout
+      client: 1860s
```

Maybe we want to align the default value to upstream. 🤔 
Ref: https://github.com/DandyDeveloper/charts/blob/redis-ha-4.33.2/charts/redis-ha/values.yaml#L224-L230

```suggestion
    timeout:
      # -- HAProxy server connection timeout
      server: 330s
      # -- HAProxy client connection timeout
      client: 330s
```